.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,10 @@
 #### What does this PR do (include background context, if relevant)?
 #### What ticket does this PR close?
-Connected to [relevant GitHub issues, eg #76]
+Connected to #[relevant GitHub issues, eg 76]
 #### Where should the reviewer start?
```

Comment:
Can se also add some newlines to separate the sections too - I keep needing to do this every time I open a PR :(